e2e/_suites/fleet/fleet.go

```diff
@@ -1300,16 +1344,21 @@ func (fts *FleetTestSuite) thePolicyIsUpdatedToHaveSystemSet(name string, set st
 	if name != "log" {
 		condition = true
 	}
+	if name == "linux/metrics" {
```

Comment:
I think you are looking for a `if/else if` case here instead of multiple `if` without `else`: 

```golang
	var condition = false
	var metrics = "system"
	var file = "/metrics.json"

	if name == "linux/metrics" {
		condition = true
		file = "/linux_metrics.json"
		metrics = "linux"
	} else if name != "system/metrics" && name != "logfile" && name != "log" {
		condition = true
	}
```
Is there any other possible value for name? If not, I'd remove the above else if, reverting the logic:

```golang
	var condition = true
	var metrics = "system"
	var file = "/metrics.json"

	if name == "linux/metrics" {
		file = "/linux_metrics.json"
		metrics = "linux"
	} else if name == "system/metrics" || name == "logfile" || name = "log" {
		condition = false
	}
```

Reply:
They all different values and can be used separate